app/app.go

```diff
@@ -455,7 +455,7 @@ func NewEvmos(
 
 	// Evmos Keeper
 	app.InflationKeeper = inflationkeeper.NewKeeper(
-		keys[inflationtypes.StoreKey], appCodec, app.GetSubspace(inflationtypes.ModuleName),
+		keys[inflationtypes.StoreKey], appCodec, authtypes.NewModuleAddress(govtypes.ModuleName).String(),
```

Comment:
Is it possible to use the type `sdk.AccAddress` for the `authority` field on the `Keeper` struct? In this way, we would not need to use the `.String()` here. I believe that would make the code cleaner. LMK your thoughts

Reply:
It is possible, I would still need to use `.String()` in the `UpdateParams` function in the message server where I check the authority matches the one from the `MsgUpdateParams` because the authority there is a string.

Reply:
Yes, at some point we have to compare the incoming string with the address